app/controllers/api/v1/rooms_controller.rb

```diff
@@ -58,6 +58,8 @@ def public_show
 
       # POST /api/v1/rooms.json
       def create
+        return render_error status: :bad_request, errors: 'RoomLimitError' if room_limit_reached(User.find(room_params[:user_id]))
```

Comment:
Can we furhter clean this `room_limit_reached(User.find(room_params[:user_id`?
We can make the method take in a `user_id` or load the user record first and maybe check if it exists before proceeding (Obscure the error in that case),